infrastructure/helm/quantumserverless/templates/keycloakrealm.yaml

```diff
@@ -61,8 +61,8 @@ data:
               ],
```

Comment:
I don't know if you can rebuild this `realm` configuration but without the `kuberay-api-server` addition. I think it could be helpful for review it, @akihikokuroda 🙏